src/plugin.js

```diff
@@ -29,11 +35,19 @@ function getResourceURL(config, filePath, branch) {
 function getEditURL(config, filePath, branch) {
     return urlJoin(`https://github.com/`, config.repo, `edit`, branch, filePath);
 }
-function getAPIURL(config, filePath) {
-    if (config["githubAPIBaseURL"]) {
-        return urlJoin(config["githubAPIBaseURL"], filePath);
+function getContentURL(config, filePath) {
+    if (config.private) {
+        return urlJoin(
+            location.origin,
+            "gitbook/gitbook-plugin-github-issue-feedback/contents",
```

Comment:
https://github.com/azu/gitbook-plugin-github-issue-feedback/pull/5/files#diff-168726dbe96b3ce427e7fedce31bb0bcR27 use `pkg.name`.
But this line use hardcode value.

We should unify usage.

1. `pkg.name`
2. hardcode
3. create shared `constants.json`

Reply:
Thank you for your review.

I'd like to choose option 2 (hard-code) in this PR. There are some other hard-coded strings. I think it is better to create another PR/Issue for refactoring them.